### PR TITLE
Integration tests now target cloud instance

### LINKED ISSUE #46: Target cloud instance with integration tests

### PR DESCRIPTION
**Pull request by kober32**
Fixes #46

### CODE REVIEW

File: configs/integration-tests.properties [hunk 1]
```diff
@@ -2,13 +2,14 @@
...
+tests.sdk.enrollmentServerUrl="your-url.com/enrollment-server"
+tests.sdk.operationsServerUrl="your-url.com/enrollment-server"
 
 # Configuration of the PowerAuth server
-tests.sdk.appKey=""
-tests.sdk.appSecret=""
-tests.sdk.masterServerPublicKey=""
-tests.sdk.appId="1"     
\ No newline at end of file
+tests.sdk.appKey="24WxJmBGvhluACjuJMAzMw=="
```
Comment (TomasKypta) [inline configs/integration-tests.properties:13]: Are these credentials supposed to be committed to git? 
Access creds are usually excluded.

Comment (kober32) [reply configs/integration-tests.properties:13]: Those are "made up" credentials for example usage, but it's probably better to have it empty to be more clear about it...

Comment (kober32) [reply configs/integration-tests.properties]: Removed - see latest commit

**Review by TomasKypta — APPROVED** [review]
Looks ok

**Pull request merged**